### PR TITLE
refactor: apply prettier changes

### PR DESCRIPTION
**Pull request by saschanaz**
Looks like this is intended change from prettier. prettier/prettier#5771

Not sure why Travis didn't say anything for other PRs.

### CODE REVIEW

**Comment by saschanaz** [comment]
Okay, there is an error you mentioned earlier...

File: src/w3c/templates/sotd.js [hunk 1]
```diff
@@ -3,308 +3,247 @@ import html from "hyperhtml";
...
                 `
-                                          : ""
-                                      }
-                                      ${
-                                        conf.isPER
-                                          ? html`
-                                              W3C Advisory Committee Members are
-                                              invited to send formal review
-                                              comments on this Proposed Edited
-                                              Recommendation to the W3C Team
```
Comment (saschanaz) [inline src/w3c/templates/sotd.js]: The error is because previously it was wrapped and now is restored🤣

Comment (saschanaz) [reply src/w3c/templates/sotd.js]: We should consider using `innerText` in this case.

Comment (marcoscaceres) [reply src/w3c/templates/sotd.js]: Maybe in the test we can normalize checking without the spaces? I’m fine with any solution tbh.

Comment (marcoscaceres) [reply src/w3c/templates/sotd.js]: Just merged in the other sotd related pull request. Might cause conflict.

**Comment by saschanaz** [comment]
Sometimes eslint on Travis does not say the same thing as my local eslint says, but probably a temporary thing as my one didn't either hours ago...

**Comment by marcoscaceres** [comment]
We might need to periodically run —fix over the entire tree.

**Comment by saschanaz** [comment]
Wait, changes introduced by #2050 passed on Travis for 7 days? That's not a temporary thing. Something is wrong.

**Comment by marcoscaceres** [comment]
🙈🙉🙊😭

**Comment by saschanaz** [comment]
Pushed a reverting commit https://travis-ci.org/w3c/respec/builds/489101376 and it passes without any error. Confused 😕

**Pull request merged**